README.md

```diff
@@ -8,6 +8,10 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/KohlsTechnology/eunomia)](https://goreportcard.com/report/github.com/KohlsTechnology/eunomia)
 [![codecov](https://codecov.io/gh/KohlsTechnology/eunomia/branch/master/graph/badge.svg)](https://codecov.io/gh/KohlsTechnology/eunomia)
 
+## Getting started
+
+If you want to deploy Eunomia from scratch to a local Minikube cluster, begin with the [Getting started](./GETTING_STARTED.md) document. It will guide your through the setup step-by-step.
+
 ## Who is Eunomia
```

Comment:
[Optional] I think we should move the `Who is Eunomia`, `What is GitOps`, and `Next Generation Change Management` sections to a different page, or at least later in the page. In my opinion the README.md should start off with what Eunomia is and what it does so the user knows as soon as they open the repo.

If we add the section for what is Eunomia/what does it do, I think it should be before the `Getting Started` section, because the user may not even know what they are starting.

This can also be done in a separate documentation cleanup effort, as I believe the issue you are currently tackling is simply a getting started guide.

Reply:
FWIW, personally, I think I'd prefer if we tried doing this in a separate PR, to keep focus. Both @sbar95's work and our reviews here are difficult enough IMO, I'm somewhat afraid of spreading our effort over multiple concerns

Reply:
You're right - but in this PR I'll just move the `Getting started` section after `Purpose`

Reply:
I've created #289 to look into cleaning up the README in a future PR.